### DISCUSSION TITLE
LogEntries integration

### DISCUSSION THREAD

**Discussion by sonicaghi** [Kong/kong#7366] [Ideas and feature requests]

**Comment by erran** [comment]
@sonicaghi The [`http-log` plugin](https://getkong.org/plugins/http-log/) can be used to forward logs to logentries.

At Rapid7 we use a configuration similar to the following:

```json
{
  "created_at": 1499338733000,
  "config": {
    "http_endpoint": "https://webhook.logentries.com/noformat/logs/LOG-TOKEN",
    "timeout": 10000,
    "method": "POST",
    "keepalive": 60000,
    "content_type": "application/json"
  },
  "id": "<generated>",
  "enabled": true,
  "api_id": "<YOUR API ID>",
  "name": "http-log"
}
```

Your `LOG-TOKEN` value can be found by following [this guide from the logentries docs](https://docs.logentries.com/v1.0/docs/http-post#section-endpoint).

@thibaultcha this issue can be closed.
